source/patterns/@aws-solutions-constructs/core/lib/elasticsearch-helper.ts

```diff
@@ -228,48 +227,63 @@ export function buildElasticSearchCWAlarms(scope: Construct): cloudwatch.Alarm[]
   return alarms;
 }
 
-function GetSubnetsAndSecurityGroups(vpc?: ec2.IVpc, props?: elasticsearch.CfnDomainProps, lambdaFunction?: lambda.Function):
+function getSubnetIds(vpc?: ec2.IVpc, props?: elasticsearch.CfnDomainProps):
   elasticsearch.CfnDomainProps {
   if (vpc) {
     // Environment specified stacks: A ES cluster deploys in 3 AZs with 3 subnets maximum(each subnet in a different AZ).
```

Comment:
"Environment specified stacks:"

I don't know what this phrase means in context